Source/ablastr/fields/PoissonSolver.H

```diff
@@ -283,19 +294,18 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
             amrex::MultiFab phi_cp(ba, phi[lev+1]->DistributionMap(), ncomp, 1);
 
             // Copy from phi[lev] to phi_cp (in parallel)
-            const amrex::IntVect& ng = amrex::IntVect::TheUnitVector();
             const amrex::Periodicity& crse_period = geom[lev].periodicity();
 
             ablastr::utils::communication::ParallelCopy(
-                phi_cp,
-                *phi[lev],
-                0,
-                0,
-                1,
-                ng,
-                ng,
+                phi_cp,                    // dst
+                *phi[lev],                 // src
+                0,                         // src_comp
+                0,                         // dst_comp
+                1,                         // num_comp
+                amrex::IntVect(1),         // src_nghost
```

Comment:
@WeiqunZhang 
I think if we change this line to  
```
amrex::IntVect(0),         // src_nghost
```
the following tests crash: `ElectrostaticSphereLabFrame_MR_emass_10`, `ElectrostaticSphereEB_RZ_MR`, `scraping` and `scraping_filter`.